test-suite/src/run-test-suite.ts

```diff
@@ -202,6 +202,17 @@ export const runTestSuite = (params: TestSuiteParams) => {
                     assert.deepEqual(await driver.$('.todo-item').attr('data-value'), '');
                 });
             });
+
+            it('should get correct attribute [when] $ is deep nested', async () => {
+              await runTest({items: [itemCreator({ label: 'Bob' })]}, async (driver) => {
+
+                  const item = await driver.$('.todo-item');
+                  assert.equal(await item.$('.label').text(), 'Bob');
```

Comment:
is this needed?

Reply:
That just making sure, but it can be removed